pillar/params.sls

```diff
@@ -95,3 +95,7 @@ components:
   proxy:
     # Extra arguments to be passed to kube-proxy.
     args: ''
+
+availability_zone:
+  region: 'unknown'
+  zone: 'unknown'
```

Comment:
I'd argue to set to them to "unknown", so that the region and zone labels appear and operators both know they exist, and have an incentive to "fix" them.